sdk/keyvault/Azure.Security.KeyVault.Administration/src/RoleAssignmentScope.cs

```diff
@@ -28,10 +28,20 @@ public RoleAssignmentScope(string value)
         /// <summary>
         /// Initializes a new instance of the <see cref="RoleAssignmentScope"/> structure.
         /// </summary>
-        /// <param name="ResourceId">The Resource Id for the given Resource</param>
+        /// <param name="ResourceId">The Resource Id for the given Resource.</param>
         public RoleAssignmentScope(Uri ResourceId)
         {
-            _value = ResourceId.AbsolutePath ?? throw new ArgumentNullException(nameof(ResourceId));
+            _ = ResourceId ?? throw new ArgumentNullException(nameof(ResourceId));
```

Comment:
Nit: Works, but more commonly I used `Argument.AssertNotNull` in Key Vault and we use in a lot of other libraries.